backend/app/GraphQL/Validators/ProfileMetadataSocialMediaValidator.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+
+namespace App\GraphQL\Validators;
+
+use Nuwave\Lighthouse\Validation\Validator;
+
+class ProfileMetadataSocialMediaValidator extends Validator
```

Comment:
Related to the similar comment on schema.graphql.  Would these be better organized inside a namespace (if the @validator directive will allow us to specify a namespace).  i.e. `class SocialMediaValidator` inside `Validators/ProfileMetadata`.

Reply:
I'll try it

Reply:
I'm not able to get this working.